src/@types/fastify/index.d.ts

```diff
@@ -0,0 +1,12 @@
+import {IncomingMessage, Server, ServerResponse} from 'http'
```

Comment:
You shouldn't have to define your own types. If types are missing from a package (such as `fastify-jwt` or `fastify-auth`) open an issue and PR on the respective projects to fix that

Reply:
I think this is needed because I use decorate, so I need to decorate the typings as well. But I can change the typings. Let me take the code and check it